metaswitch/common/alarms_parser.py

```diff
@@ -30,37 +30,41 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 import json
+import alarm_severities
 
-# Valid severity levels - this should be kept in sync with the 
+# valid severity levels - this should be kept in sync with the
 # list in alarmdefinition.h in cpp-common
-# Alarms are stored in ITU Alarm Table using the severities below.
-# Alarm Model Table stores alarms according to their state. The
-# mapping between state and severity is described in RFC 3877
+# alarms are stored in itu alarm table using the severities below.
+# alarm model table stores alarms according to their state. the
+# mapping between state and severity is described in rfc 3877
 # section 5.4: https://tools.ietf.org/html/rfc3877#section-5.4
-# The function AlarmTableDef::state() maps severities to states.
+# the function alarmtabledef::state() maps severities to states.
```

Comment:
I'm happy for the function name to remain uncapitalised if that is how it is in the code. Where is this function?